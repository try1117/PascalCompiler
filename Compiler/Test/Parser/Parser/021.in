program test;
var
  a : record
    a : integer;
    b : double;
    c : array [-5..5] of char;
  end;

begin
end.
