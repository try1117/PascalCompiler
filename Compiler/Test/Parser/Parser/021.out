test : function()
   resultType : Nil
   
Declarations:
   a : Record
      a : Integer
      b : Double
      c : Array [-5, 5] of Char
   end
